sqlserver/datadog_checks/sqlserver/const.py

```diff
@@ -93,6 +93,11 @@
     # SQLServer:Plan Cache
     ('sqlserver.cache.object_counts', 'Cache Object Counts', '_Total'),
     ('sqlserver.cache.pages', 'Cache Pages', '_Total'),
+    # SQLServer:Transactions
+    ('sqlserver.stats.version_store_size', 'Version Store Size (KB)', ''),
+    ('sqlserver.stats.version_cleanup_rate', 'Version Cleanup rate (KB/s)', ''),
+    ('sqlserver.stats.version_generation_rate', 'Version Generation rate (KB/s)', ''),
+    ('sqlserver.stats.longest_transaction_running_time', 'Longest Transaction Running Time', ''),
```

Comment:
Should/could we have a test for those? I'm not actually sure what our test coverage is for those `INSTANCE_METRICS` so the answer might be that it's not necessary for tests specific to the new metrics but I thought I'd ask.

Reply:
They are covered in [test_check_docker](https://github.com/DataDog/integrations-core/blob/097a9e407070f15263f602157bca4f9c7c91f431/sqlserver/tests/test_integration.py#L53C1-L53C1) under [assert_metrics](https://github.com/DataDog/integrations-core/blob/097a9e407070f15263f602157bca4f9c7c91f431/sqlserver/tests/common.py#L237)
(`assert_metrics` internally check `EXPECTED_METRICS` which contains the `INSTANCE_METRICS`)